README.md

```diff
@@ -43,13 +43,7 @@ Thanks goes to these wonderful people
om/all-contributors/all-contributors-cli/commits?author=tigermarques" alt="Documentation">📖</a> <a href="#ideas-tigermarques" alt="Ideas, Planning, & Feedback">🤔</a></td><td><a href="http://hipstersmoothie.com"><img src="https://avatars3.githubusercontent.com/u/1192452?v=4" width="100px;" alt="Andrew Lisowski"/><br><sub><b>Andrew Lisowski</b></sub></a><br><a href="https://github.com/all-contributors/all-contributors-cli/commits?author=hipstersmoothie" alt="Code">💻</a> <a href="https://github.com/all-contributors/all-contributors-cli/commits?author=hipstersmoothie" alt="Documentation">📖</a> <a href="https://github.com/all-contributors/all-contributors-cli/commits?author=hipstersmoothie" alt="Tests">⚠️</a></td><td><a href="https://github.com/chinesedfan"><img src="https://avatars3.githubusercontent.com/u/1736154?v=4" width="100px;" alt="Xianming Zhong"/><br><sub><b>Xianming Zhong</b></sub></a><br><a href="https://github.com/all-contributors/all-contributors-cli/commits?author=chinesedfan" alt="Documentation">📖</a></td><td><a href="https://github.com/xuchaoying"><img src="https://avatars2.githubusercontent.com/u/8073251?v=4" width="100px;" alt="C.Y.Xu"/><br><sub><b>C.Y.Xu</b></sub></a><br><a href="https://github.com/all-contributors/all-contributors-cli/commits?author=xuchaoying" alt="Code">💻</a></td></tr><tr><td><a href="https://github.com/chris-dura"><img src="https://avatars3.githubusercontent.com/u/3680914?v=4" width="100px;" alt="Dura"/><br><sub><b>Dura</b></sub></a><br><a href="https://github.com/all-contributors/all-contributors-cli/commits?author=chris-dura" alt="Documentation">📖</a></td><td><a href="https://jakebolam.com"><img src="https://avatars2.githubusercontent.com/u/3534236?v=4" width="100px;" alt="Jake Bolam"/><br><sub><b>Jake Bolam</b></sub></a><br><a href="#infra-jakebolam" alt="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="https://github.com/all-contributors/all-contributors-cli/commits?author=jakebolam" alt="Code">💻</a> <a href="https://github.com/all-contributors/all-contributors-cli/commits?author=jakebolam" alt="Documentation">📖</a> <a href="https://github.com/all-contributors/all-contributors-cli/commits?author=jakebolam" alt="Tests">⚠️</a></td><td><a href="http://maxcubing.wordpress.com"><img src="https://avatars0.githubusercontent.com/u/8260834?v=4" width="100px;" alt="Maximilian Berkmann"/><br><sub><b>Maximilian Berkmann</b></sub></a><br><a href="https://github.com/all-contributors/all-contributors-cli/commits?author=Berkmann18" alt="Code">💻</a> <a href="https://github.com/all-contributors/all-contributors-cli/commits?author=Berkmann18" alt="Tests">⚠️</a> <a href="https://github.com/all-contributors/all-contributors-cli/commits?author=Berkmann18" alt="Documentation">📖</a></td><td><a href="https://github.com/tbenning"><img src="https://avatars2.githubusercontent.com/u/7265547?v=4" width="100px;" alt="tbenning"/><br><sub><b>tbenning</b></sub></a><br><a href="#design-tbenning" alt="Design">🎨</a></td></tr></table>
```

Comment:
Github is rendering like this:
(note the `<style>` tags)
<img width="903" alt="screen shot 2019-02-02 at 4 53 32 pm" src="https://user-images.githubusercontent.com/3534236/52169719-1d3a6300-270b-11e9-95d9-b97bf901ad1c.png">

Reply:
@Berkmann18 this is the only.blocking comment

Reply:
Yeah.